charts/nm-vllm-grafana-dashboards/templates/config-map-list.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: ConfigMapList
+items:
+{{- $files := .Files.Glob "dashboards/*.json" }}
+{{- range $path, $fileContents := $files }}
+{{- $dashboardName := regexReplaceAll "(^.*/)(.*)\\.json$" $path "${2}" }}
+- apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: {{ template "nm-vllm-grafana-dashboards.fullname" $ }}-grafana-dashboard
+    labels:
+      {{- if $.Values.label }}
+      {{ $.Values.label }}: {{ ternary $.Values.labelValue "1" (not (empty $.Values.labelValue)) | quote }}
+      {{- end }}
+  data:
+    {{ $dashboardName }}.json: {{ $.Files.Get $path | toJson }}
```

Comment:
Maybe we should keep the `.json` on the regexp replace so it doesn't need to be readded here? If we add a label identifying the dashboard, I'd say readding `.json` here is fine, but I don't think we use `$dashboardName` anywhere else.

Reply:
Yeah I think in the prometheus grafana chart they ended up also using the dashboardName as a backup name or something for the metadata name, so I guess I should check if that is something we should do

Reply:
Oh yeah, that's a fantastic point. Right now any additional ConfigMaps would end up with the same name and I can't imagine that'd work out well.

Reply:
I've fixed the naming issue, and as such it is now more obvious that we should extract the dashboard name without the `.json` suffix.